.gitignore

```diff
@@ -1,3 +1,7 @@
+# folder
 .idea
 venv/
 __pycache__/
+
+#file
+*.pyc
```

Comment:
I suspect that this newline at EOF is causing the build to fail on TravisCI, I will look into it in a further commit for this PR.